terraform/modules/region/cloudwatch.tf

```diff
@@ -16,6 +16,126 @@ resource "aws_kms_alias" "cloudwatch_standard_alias" {
   target_key_id = aws_kms_key.cloudwatch_standard.key_id
 }
 
+resource "aws_cloudwatch_log_metric_filter" "uid_service_400_errors" {
+  name           = "${local.environment_name}-uid-service-400-errors"
+  pattern        = "$.status = 400"
```

Comment:
I had a little play with the [metric filter UI](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logsV2:log-groups/log-group/$252Fecs$252Fdev/create-metric-filter) which lets you put test a filter against some custom data, and found that you need to wrap the statement in curlys: `{ $.status = 400 }`.

Similarly, matching a message requires curlys and some extra wildcards: `{$.message = "*/source*required*"}`

But that said, neither of these will actually work because the logging is currently outputting errors like this and doesn't log problem details:
```
{"service_name":"opg-data-lpa-uid","timestamp":"2023-04-17T09:53:38.288479187Z","message":"error unmarshalling json content"}
```

So I think you also need to add something to `Problem.Respond()` that logs the problem. opg-go-common logger doesn't really support this, so you'll probably need to do it more manually, something like:
```
type LogEvent struct {
  ServiceName string    `json:"service_name"`
  Timestamp   time.Time `json:"timestamp"`
  Status      int       `json:"status"`
  Problem     Problem   `json:"problem"`
}

json.NewEncoder(os.Stdout).Encode(LogEvent{
  ServiceName: "opg-data-lpa-uid",
  Timestamp:   time.Now(),
  Status:      problem.StatusCode,
  Problem:     problem,
})
```
Which should produce error messages like:
```
{"service_name":"opg-data-lpa-uid","timestamp":"2023-04-17T10:43:29.584580045Z","status":400,"problem":{"code":"INVALID_REQUEST","detail":"Invalid request","errors":[{"source":"/source","detail":"required"}]}}
```